www/app/App.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import BaseApp from 'next/app';
+import Head from 'next/head';
+import keyboardOnlyOutlines from 'keyboard-only-outlines';
+import ReactGA from 'react-ga';
+import getConfig from 'next/config';
+
+import seoImage from '../shared/media/images/seo.png';
+import favicon from '../shared/media/favicons/favicon.ico';
+
+import '../shared/styles/index.css';
+
+const { publicRuntimeConfig } = getConfig();
+
+const SEO_DATA = {
+    url: 'https://{project-domain}',
+    title: '{project-name}',
+    description: '{project-description}',
+    keywords: ['{project-keyword-1}', '{project-keyword-2}'],
+    image: { src: seoImage, width: 1200, height: 630 },
+};
+
+class App extends BaseApp {
+    componentDidMount() {
+        keyboardOnlyOutlines();
+
+        // Initialize Google Analytics
+        if (publicRuntimeConfig.NEXT_PUBLIC_GA_TRACKING_ID) {
+            ReactGA.initialize(publicRuntimeConfig.NEXT_PUBLIC_GA_TRACKING_ID);
+            ReactGA.pageview(window.location.pathname + window.location.search);
+        }
+    }
```

Comment:
I would use the Google Analytics integration like this to ensure it would trigger on every route change:

```js
componentDidMount() {
    keyboardOnly();

    if (NEXT_PUBLIC_GA_TRACKING_ID) {
        ReactGA.initialize(NEXT_PUBLIC_GA_TRACKING_ID);
        ReactGA.pageview(router.asPath);

        this.props.router.events.on('routeChangeComplete', this.handleRouteChange);
    }
}

componentWillUnmount() {
    if (NEXT_PUBLIC_GA_TRACKING_ID) {
        this.props.router.events.off('routeChangeComplete', this.handleRouteChange);
    }
}

// ...

handleRouteChange = (url) => {
    if (NEXT_PUBLIC_GA_TRACKING_ID) {
        ReactGA.pageview(url);
    }
};
```